### ISSUE TITLE
Getting "`key` is not a prop" after updating to 6.0.3

### ISSUE THREAD

**Issue by Linksku** [welldone-software/why-did-you-render#185]
I was on 6.0.5 previously. After updating to 6.1.0, I'm getting the following 2 warnings:

`react_devtools_backend.js:2430 Warning: [object Object]: `key` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://reactjs.org/link/special-props)`

`react_devtools_backend.js:2430 Warning: div: `ref` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://reactjs.org/link/special-props)`

It's just a React devtool warning, but it's still annoying. It appears on components that aren't even using `key` or `ref`.

Here's how I'm using wdyr:

```
if (process.env.NODE_ENV !== 'production') {
  // eslint-disable-next-line global-require, import/no-extraneous-dependencies
  const whyDidYouRender = require('@welldone-software/why-did-you-render');
  whyDidYouRender(React, {
    trackAllPureComponents: true,
  });
}
```

**Comment by Linksku** [comment]
I just noticed v6.1.1+ is released already, I don't know why Yarn gave me 6.0.3. I'll try getting 6.1.1.

**Comment by Linksku** [comment]
Just noticed 6.1.1 is released, that seems to have fixed the issue.
